src/commands/create-unik.ts

```diff
@@ -40,20 +35,14 @@ export class CreateUnikCommand extends WriteCommand {
         return CreateUnikCommand;
     }
 
-    protected async do(flags: Record<string, any>): Promise<NestedCommandOutput> {
+    protected async do(flags: Record<string, any>): Promise<any> {
```

Comment:
Why the type `NestedCommandOutput` has disappeared?
Returned object as the same signature as before